packages/sortable-behavior-bundle/README.md

```diff
@@ -32,6 +32,31 @@ return [
 ];
 ```
 
+### Use a draggable list instead of up/down buttons
```

Comment:
I think we should make another section like 

```suggestion
## Usage

...(Here we should explain how to use the bundle with default options)

...(Show full config options)

...(Show alternative like the drag and drop)

### Use a draggable list instead of up/down buttons
```